core/arch/arm/kernel/thread_spmc.c

```diff
@@ -220,7 +224,7 @@ static void unmap_buf(void *va, size_t sz)
 	tee_mm_free(mm);
 }
 
-static void handle_rxtx_map(struct thread_smc_args *args)
+void spmc_handle_rxtx_map(struct thread_smc_args *args, struct ffa_rxtx *buf)
```

Comment:
I'd prefer `rxtx` instead of `buf` as the name of that parameter.

Reply:
Done

Reply:
Done

Reply:
Please fix this in the other functions here too.

Reply:
done